packages/vuetify/src/framework.ts

```diff
@@ -14,59 +19,88 @@ import {
 // Services
 import * as services from './services'
 
-export default class Vuetify {
-  static install = install
+export const VuetifySymbol = Symbol.for('vuetify')
 
-  static installed = false
+export function useVuetify () {
+  const vuetify = inject(VuetifySymbol)
+
+  /* istanbul ignore if */
+  if (!vuetify) {
+    warn('Vuetify has not been installed on this app')
+  }
 
-  static version = __VUETIFY_VERSION__
+  return vuetify
+}
+
+export const version = __VUETIFY_VERSION__
+
+export default class Vuetify {
+  public defaultPreset = Preset
 
   public framework: Dictionary<VuetifyServiceContract> = {}
 
-  public installed: string[] = []
+  public installed: Set<string> = new Set()
 
   public preset = {} as VuetifyPreset
 
   public userPreset: UserVuetifyPreset = {}
```

Comment:
this is not used

Reply:
Used here https://github.com/vuetifyjs/vuetify/blob/refactor/integrate-vue-3-alpha/packages/vuetify/src/framework.ts#L100